clients/extension/styles/lighthouse.css

```diff
@@ -50,6 +50,13 @@ body {
   margin: 2px auto 0;
 }
 
+.browser-brand {
+  display: none;
+}
+.browser-brand--__MSG_browserBrand__ {
```

Comment:
Is `__MSG_browserBrand__` some magic from extensions that pipes through localized messages? Seems like it should be handled in js and not piped in via some CSS :man_shrugging:

Reply:
https://developer.chrome.com/extensions/i18n

I rather like this.

Reply:
I don't like the idea of overloading the locale system to populate this, what is essentially, a build time env var. It seems like an abuse of the system.

Reply:
1) we don't even have plans to localize this extension. maybe we could refactor it later if we do?
2) what else could we do?

Reply:
done